cltk/corpus/arabic/corpora.py

```diff
@@ -0,0 +1,9 @@
+"""Available Arabic corpora available for download."""
+
+ARABIC_CORPORA = [
+    {'name': 'Tashkeela2_Arabic_Vocalized_text',
+     'markup':'xml',
+     'origin': 'git@github.com:LBenzahia/tashkeela2.git',
```

Comment:
We use http urls, not the git ones. Thus this would be `https://github.com/LBenzahia/tashkeela2.git`